src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -605,4 +651,14 @@ private static MablRestApiClient createMablRestApiClient(
                 disableSslVerification);
     }
 
+    private void updateBrowser(String browserName , boolean selected){
+        if(selected){
+            if(browsers == null) browsers = new ArrayList<>();
+            if(!browsers.contains(browserName)) browsers.add(browserName);
+        }
+        else if(browsers != null){
```

Comment:
We have `null` checks here for `this.browsers`, but it's initialized at class creation to an empty list. Is there a situation where it would be `null`?